scipy/stats/morestats.py

```diff
@@ -2692,15 +2692,13 @@ def circmean(samples, high=2*pi, low=0, axis=None):
 
     """
     samples, ang = _circfuncs_common(samples, high, low)
-    res = angle(np.mean(exp(1j * ang), axis=axis))
-    mask = res < 0
+    S = sin(ang).mean(axis=axis)
+    C = cos(ang).mean(axis=axis)
```

Comment:
Updated. Thanks! 
Anyway, I was surprised to see np.mean() performing poorer than np.sum()/len(). Are you aware of any reason why it is the case?